### ISSUE TITLE
Provide an example of the library usage with google

### ISSUE THREAD

**Issue by jadekler** [jadekler/spring-security-saml-dsl#4]
Along with a README that appropriately describes the steps to setup okta.

See: https://developers.google.com/google-apps/sso/saml_reference_implementation

**Comment by markthemarkest** [comment]
It looks to me like those docs are to use Google as a Service Provider when working with another Identity Provider.

**Comment by rwinch** [comment]
@markthemarkest You are right. I think a better link might be https://support.google.com/a/answer/6087519?hl=en

**Comment by markthemarkest** [comment]
Ah that makes more sense, thanks. However, I don't believe that I have access to any admin console where I could set something like that up. Google doesn't seem to provide free developer instances to test integrations, unless I've missed something.

**Comment by rwinch** [comment]
@markthemarkest I'm not sure that they do. Perhaps @ttomsu knows if Google provides a free SAML based IdP?

**Comment by ttomsu** [comment]
Not that I know of - it appears Google Apps for Work has [some SAML support](https://support.google.com/a/answer/6087519?hl=en), but I haven't tried it yet.

**Comment by rwinch** [comment]
@ttomsu Thanks for the response. Do you need SAML integration with Google as well or do you just need SAML integration with Okta?

**Comment by ttomsu** [comment]
I only used Okta because there was a free developer preview. In theory, since it's based on the standard, any SAML IdP should work. I'll try to get the Google SAML one working as a proof-of-concept.

**Comment by ttomsu** [comment]
Looks like one can setup SAML support through the Admin console --> Security --> Set up single sign-on (SSO).


![googlesaml2](https://cloud.githubusercontent.com/assets/13141550/15330522/3543e432-1c2b-11e6-87ff-a1aa078ae4fe.png)

**Comment by swarm1** [comment]
Does anybody has any example how to use it with google ? I followed example for Okta but I am getting:

 Error: app_not_configured_for_user
Service is not configured for this user.

**Comment by ttomsu** [comment]
It sounds like you need to enable it for all or a subset of your users.

1. From the Admin Console, click Apps --> SAML Apps --> your_app_name
2. In the block with your app name, you should see three vertical dots with 3 options:
  1. ON for everyone
  2. OFF
  3. ON for some organizations

Click one of the ON options to enable.
